extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/sql_load_scirpt/DefaultSqlLoadScriptTestCase.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.hibernate.orm.sql_load_scirpt;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DefaultSqlLoadScriptTestCase {
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addAsResource("application.properties"));
+
+    @Test
+    public void testDefaultSqlLoadScriptTest() {
+    }
```

Comment:
I think this should test something so to make sure that that specific configuration resource was applied, and we also need to make sure that it all actually booted successfully?

Reply:
Yes. I was thinking of persisting an entity using the load script and make sure that using a `Entity.findById()` returns something.   Will this be enough in this case?

Reply:
right I think that would be great.

Reply:
Assertions are that the script is configuration is applied.